algorithms/kernel/dtrees/gbt/regression/gbt_regression_train_dense_default_impl.i

```diff
@@ -55,26 +55,35 @@ public:
     virtual void getGradients(size_t n, size_t nRows, const algorithmFPType * y, const algorithmFPType * f, const IndexType * sampleInd,
                               algorithmFPType * gh) DAAL_C11_OVERRIDE
     {
-        if (sampleInd)
-        {
-            PRAGMA_IVDEP
-            PRAGMA_VECTOR_ALWAYS
-            for (size_t i = 0; i < n; ++i)
+        const size_t nThreads  = daal::threader_get_threads_number();
+        const size_t nBlocks   = getNBlocksForOpt<cpu>(nThreads, n);
+        const size_t nPerBlock = n / nBlocks;
+        const size_t nSurplus  = n % nBlocks;
+        const bool inParallel  = nBlocks > 1 ? true : false;
```

Comment:
See above

Reply:
Fixed

Reply:
@averbukh I applied your comments. Can you mark requested changes as made?